samples/modules/lvgl/screen_transparency/src/main.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2024 Martin Stumpf <finomnis@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/display.h>
+
+#include <lvgl.h>
+#include <stdio.h>
+#include <string.h>
+#include <zephyr/kernel.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(app, CONFIG_LOG_DEFAULT_LEVEL);
+
+static void initialize_gui(void)
+{
+	lv_obj_t *label;
+
+	/* Configure screen and background for transparency */
+	lv_obj_set_style_bg_opa(lv_scr_act(), LV_OPA_TRANSP, LV_PART_MAIN);
+	lv_disp_set_bg_opa(NULL, LV_OPA_TRANSP);
+	lv_obj_set_style_bg_color(lv_scr_act(), lv_color_hex(0x000000), LV_PART_MAIN);
+
+	/* Create a label, set its text and align it to the center */
+	label = lv_label_create(lv_scr_act());
+	lv_label_set_text(label, "Hello, world!");
+	lv_obj_set_style_text_color(lv_scr_act(), lv_color_hex(0xff00ff), LV_PART_MAIN);
```

Comment:
shouldn't this be 
```suggestion
	lv_obj_set_style_text_color(label, lv_color_hex(0xff00ff), LV_PART_MAIN);
```

Reply:
That's copy paste from https://github.com/lvgl/lvgl/blob/master/examples/get_started/lv_example_get_started_1.c, but, maybe? I'm not fully familiar with LVGL yet, I'm only setting up the tooling for it for our project right now.

Reply:
But I would probably keep it as close as possible to the lvgl hello world example